libimage/load.go

```diff
@@ -24,15 +24,25 @@ type LoadOptions struct {
 
 // doLoadReference does the heavy lifting for LoadReference() and Load(),
 // without adding debug messages or handling defaults.
-func (r *Runtime) doLoadReference(ctx context.Context, ref types.ImageReference, options *LoadOptions) (images []string, transportName string, err error) {
+func (r *Runtime) doLoadReference(ctx context.Context, ref types.ImageReference, options *LoadOptions) (names []string, transportName string, err error) {
```

Comment:
When you have had to figure it out, please document what are the exact semantics of the returned strings.